docs/source/interfaces/Engineering_Diffraction.rst

```diff
@@ -100,6 +116,30 @@ numbers. For example::
    2, 100, 102, 107
    3, 300, 310, 320-329, 350-370
 
+When a focus run process is being carried out, Focus Stop button will
+be enabled. Focus Stop button will allow the user to abort once the
+current focus run process has been completed. Inside the *Result Log*
+a warning message will be displayed with last successful run and total
+number of focus runs that could not be processed.
+
+Run Number
+^^^^^^^^^^
+The run provided to focus can be for example 228061-228063, this will
+run all the files within the given range as long as the file
+directories are included in the
+`User Directories <http://www.mantidproject.org/SplittersWorkspace>`_.
+The user may also provide an input of 228061-3 or 228061, 228062,
+2280623 which should work the same way.
+
+If a red star sign is displayed next to the Browse Button, it is mostly
+likely because the file specified has not been found. Error message
+can be viewed by hovering over the red star sign.
+
+It is also possible sometimes that a file has not been found but
```

Comment:
I'd explain here why this happens. Something slong the lines of "checking the availability of all the files can take some time. For this reason..."